values.yaml

```diff
@@ -188,6 +188,14 @@ server:
   # See https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   updateStrategyType: "OnDelete"
 
+  # Configure the logging verbosity for the Vault server.
+  # Supported log levels include: trace, debug, error, warn, info
+  logLevel: "info"
```

Comment:
In order of precedence, Vault prefers `Flags > Env var > Config file`[1], so defaulting this to "info" may surprise some folks who have log level set in their config file. Same goes for logFormat of course.

[1]: https://www.vaultproject.io/docs/configuration#log_level

Reply:
Error is the least verbose, then warn then info, then debug and trace is the most verbose.

Reply:
I kinda like the idea of defaulting to empty, and only setting the logLevel env var if this has been set by the user, since this is something that could also be set in the vault config file.

Reply:
Good point @tvoran, I added that.